src/GRA.Web/Areas/MissionControl/Views/PerformerManagement/PerformerDetails.cshtml

```diff
@@ -1,5 +1,7 @@
 ﻿@model GRA.Controllers.ViewModel.MissionControl.PerformerManagement.PerformerDetailsViewModel
 
+@inject Microsoft.Extensions.Localization.IStringLocalizer<GRA.Resources.Shared> SharedLocalizer
```

Comment:
Localization isn't needed on the MissionControl side of things.

Reply:
Fixed in commit a644d762f46ed05c11f0f643e52b135003a8f354

Reply:
Fixed in commit 97b7d4757c5daf38b97e01a4b0f363bb290ee169